### PR TITLE
[ProductImages] - Improve CSS Modules

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

Improve the handles to allow customization with css modules.  

#### How should this be manually tested?
https://productimage--storecomponents.myvtex.com/classic-shoes/p

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed 
+- Improve css handles in `ProductImages` component.
```
Comment (klzns) [inline CHANGELOG.md]: ⛏ 
```suggestion
- Improve CSS handles in `ProductImages` component. 
```

File: react/components/ProductImages/components/BlurredLoader/index.js [hunk 2]
```diff
@@ -120,33 +121,35 @@ class BlurredLoader extends React.Component {
 
     return (
       <React.Fragment>
```
Comment (klzns) [inline react/components/ProductImages/components/BlurredLoader/index.js]: Remove unnecessary Fragment

File: react/components/ProductImages/components/BlurredLoader/index.js [hunk 3]
```diff
@@ -120,33 +121,35 @@ class BlurredLoader extends React.Component {
...
-              minRatio={imageMinRatio}
-              className={`absolute z-1 w-100 center left-0 right-0 bottom-0 top-0 transition-opacity-1 db ${
-                loadState === LOAD_STATES.LOADING ? 'o-0' : 'o-100'
-              }`}
-            />
-          </div>
-        )}
+        <div className={styles.image}>
+          <ImageResizer
+            className={`w-100 ${loaded ? 'db' : 'dn'}`}
```
Comment (JuanAlmeida12) [inline react/components/ProductImages/components/BlurredLoader/index.js]: here too

File: react/components/ProductImages/components/BlurredLoader/index.js [hunk 4]
```diff
@@ -120,33 +121,35 @@ class BlurredLoader extends React.Component {
...
+                className={`w-100 ${styles.imageBlur30} ${styles.imageTransitionOpacity} db z-2 ${
+                  loadState === LOAD_STATES.LOADING ? 'o-100' : 'o-0'
+                  } ${className}`}
+              />
+              <ImageResizer
+                alt=""
+                src={realUrls[realUrlIndex]}
+                minRatio={imageMinRatio}
+                className={`absolute z-1 w-100 center left-0 right-0 bottom-0 top-0 ${styles.imageTransitionOpacity} db ${
+                  loadState === LOAD_STATES.LOADING ? 'o-0' : 'o-100'
```
Comment (JuanAlmeida12) [inline react/components/ProductImages/components/BlurredLoader/index.js]: here too

File: react/components/ProductImages/components/BlurredLoader/index.js [hunk 5]
```diff
@@ -80,19 +81,19 @@ class BlurredLoader extends React.Component {
       case LOADER_TYPES.LINEAR:
         return (
           <div
-            className={`w-100 top-0 z-2 absolute transition-opacity-1 ${
+            className={`w-100 top-0 z-2 absolute ${styles.imageTransitionOpacity} ${
               loadState === LOAD_STATES.LOADING ? 'o-100' : 'o-0'
```
[the remark this answers is not in the document]
Comment (ArthurSampaio) [reply react/components/ProductImages/components/BlurredLoader/index.js]: I dont see improvement with this tip. I dont did it, but this ternary operation is used to choose between 2 classes, if i use classNames i should do something like this 
```  
const defaultClass = classNames(..., {'o-100': loadState === LOAD_STATES.LOADING, 
           'o-0': loadState !== LOAD_STATE.LOADING,
}

```


I really think that classNames dont improve any technical aspects. Instead, worse the code legility. What do you think?

[the remark this answers is not in the document]
Comment (JuanAlmeida12) [reply react/components/ProductImages/components/BlurredLoader/index.js]: You can put this `loadState !== LOAD_STATE.LOADING` in a constant `isLoading` and use this 
```js
const isLoading = loadState !== LOAD_STATE.LOADING
const defaultClass = classNames(..., {
  'o-100': isLoading,
  'o-0': !isLoading
}
``` 
I think the code will be more prettier and legible, what you think?

[the remark this answers is not in the document]
Comment (ArthurSampaio) [reply react/components/ProductImages/components/BlurredLoader/index.js]: i disagree dude, rsrs. but i will do this it's a small and easy change. :hammer:

[the remark this answers is not in the document]
Comment (ArthurSampaio) [reply react/components/ProductImages/components/BlurredLoader/index.js]: done! :hammer:

File: react/components/ProductImages/components/BlurredLoader/index.js [hunk 6]
```diff
@@ -80,19 +81,19 @@ class BlurredLoader extends React.Component {
...
           >
             <LinearProgressWithStyle className="c-action-primary" />
           </div>
         )
       case LOADER_TYPES.SPINNER:
         return (
           <div
-            className={`absolute z-2 center-all left-0 right-0 top-0 bottom-0 transition-opacity-1 ${
+            className={`absolute z-2 center-all left-0 right-0 top-0 bottom-0 ${styles.imageTransitionOpacity} ${
               loadState === LOAD_STATES.LOADING ? 'o-100' : 'o-0'
```
[the remark this answers is not in the document]
Comment (ArthurSampaio) [reply react/components/ProductImages/components/BlurredLoader/index.js]: done! :hammer:

File: react/components/ProductImages/components/BlurredLoader/index.js (hunk 4 above)
Comment (ArthurSampaio) [reply react/components/ProductImages/components/BlurredLoader/index.js]: done! :hammer:

**Pull request merged**